livekit-rtc/livekit/rtc/room.py

```diff
@@ -436,6 +439,28 @@ async def _listen_task(self) -> None:
             self._room_queue.put_nowait(event)
             await self._room_queue.join()
 
+    def _on_rpc_method_invocation(self, rpc_invocation: RpcMethodInvocationEvent):
+        if self._local_participant is None:
+            logging.warning(
+                "Received RPC invocation before local participant was initialized"
+            )
+            return
```

Comment:
you can safely remove this code

Reply:
actually i couldn't remove this because the type checker complains about accessing properties if its none. i removed the log line but kept the check